### DISCUSSION TITLE
Switching version does not work on my MacBookPro

### DISCUSSION THREAD

**Discussion by thommy66** [pyenv/pyenv#2228] [Q&A] [answered]
I run MacOS BigSur with an intel mac and installed pyenv using `brew`. I mainly use it for [Ansible](https://www.ansible.com/) to deploy software, containers etc.

Here is the situation which means that it is using **always** 2.7.16, no matter what I set using `pyenv global` or `pyenv local`:
```
[bin] python -V
Python 2.7.16
[bin] pyenv versions
  system
* 3.10.1 (set by /Users/tbednarz/.pyenv/version)
```

My brew installation does not report any problems:
```
[bin] brew doctor
Your system is ready to brew.
```
And this is what I have installed regarding python:
```
[bin] brew list | grep py
pyenv
python@3.10
python@3.9
```

Interesting also that brew shows V3.9 and V3.10 but pyenv only sees `system` and `3.10.1`. 

Is there anything messed up on my system.

**Comment by jayknowstheway** [comment]
where are the brilliant engineers that have an answer to this?

**Comment by joshfriend** [comment, marked as answer]
If you have a problem, file an issue and follow the issue template for basic troubleshooting steps.
